ProgrammingTask/network.py

```diff
@@ -4,14 +4,31 @@
 """
 
 import socket
-
+import pickle
 from config import SERVER_ADDRESS, SERVER_PORT
 
 
 class Network:
-    def create_connection(self):
-        print("connecting to server...")
+    def __init__(self):
+        self.client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.server = SERVER_ADDRESS
+        self.port = SERVER_PORT
+        self.addr = (self.server, self.port)
+        self.playerId = self.connect()
+
+    def getPlayerId(self):
+        return self.playerId
+
+    def connect(self):
+        try:
+            self.client.connect(self.addr)
+            return self.client.recv(2048).decode()
+        except:
```

Comment:
I'd love to have all the `except` -blocks with some specific target exceptions they are supposed to catch. The catch-all except is problematic since it will catch unexpected exceptions like typos too.

So instead of `except:` we would have `except SomeCertainException:` and so on.